### PR TITLE
Introduce process.vpid

### PR DESCRIPTION
**Pull request by Tacklebox**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/ecs/blob/main/CONTRIBUTING.md)?
- For proposing substantial changes or additions to the schema, have you reviewed the [RFC process](https://github.com/elastic/ecs/blob/main/rfcs/README.md)?
- If submitting code/script changes, have you verified all tests pass locally using `make test`?
- If submitting schema/fields updates, have you generated new artifacts by running `make` and committed those changes?
- Is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- Have you added an entry to the [CHANGELOG.next.md](https://github.com/elastic/ecs/blob/main/CHANGELOG.next.md)?

### CODE REVIEW

**Comment by kgeller** [comment]
@Tacklebox the change to the schema file looks good, but you'll want to run `make` to generate your change in the artifacts

**Review by mitodrummer — APPROVED** [review]

**Comment by Tacklebox** [comment]
@kgeller When I run `make` git shows there are no changes in the generated files. I tried `make clean && make`, and my working tree is still up to date with no changes. Do I need to update any of the other files to have the new field included?

**Comment by kgeller** [comment]
@Tacklebox Apologies, yes, you'll want to update the [subset file](https://github.com/elastic/ecs/blob/main/schemas/subsets/main.yml). For certain fieldsets where we want to better nest fields, we list out where we expect what fields. You'll want to add the `vpid` field everywhere you expect it to be (ie under regular `process.*` or `process.parent.*` etc) and then re-run `make`

Let me know if you run into any issues

**Review by kgeller — APPROVED** [review]
LGTM

As long as @norrietaylor and @mitodrummer agree those are all the spots we want `vpid` to be nested at. If we're unsure, we can always release the field as beta for a release. But I will defer to you all.

**Review by mitodrummer — APPROVED** [review]
LGTM!

**Comment by norrietaylor** [comment]
LGTM!

File: docs/fields/field-details.asciidoc [hunk 1]
```diff
@@ -8593,6 +8593,24 @@ example: `1325`
 
 // ===============================================================
 
+|
+[[field-process-vpid]]
+<<field-process-vpid, process.vpid>>
+
+a| Virtual Process id.
+
+The process id within a cgroup pid namespace. This is not necessarily unique across all processes on the host but it is unique within the process namespace that the process exists within.
```
Comment (mmat11) [inline docs/fields/field-details.asciidoc:8602]: > The process id within a cgroup pid namespace.

Shouldn't this be: `The process id within a pid namespace.`?

File: docs/fields/field-details.asciidoc [hunk 2]
```diff
@@ -8593,6 +8593,24 @@ example: `1325`
 
 // ===============================================================
 
+|
+[[field-process-vpid]]
+<<field-process-vpid, process.vpid>>
+
+a| Virtual Process id.
```
Comment (mmat11) [inline docs/fields/field-details.asciidoc:8600]: nit, for consistency

```suggestion
a| Virtual process id.
```

**Review by mmat11 — APPROVED** [review]
lgtm!

**Pull request merged**